Documentation/Signoffs/OEM Component- Auto_Light-signoff.md

```diff
@@ -0,0 +1,67 @@
+OEM Component Subsystem- Auto Light
```

Comment:
I don't understand why both sets of schematics are here. Which is the design?

Analysis showing that the components meet the requirements of the circuit is not present.

Reply:
I just added one schematic, and that is my design.